docs/containers/bridge-to-kubernetes.md

```diff
@@ -20,141 +20,67 @@ In this guide, you will learn how to use Bridge to Kubernetes to redirect traffi
-
 ## Connect to your cluster and debug a service
 
-On your development computer, download and configure the Kubernetes CLI to connect to your Kubernetes cluster using [az aks get-credentials][az-aks-get-credentials].
-
-```azurecli
-az aks get-credentials --resource-group MyResourceGroup --name MyAKS
-```
-
-Open `mindaro/samples/BikeSharingApp/Bikes` from the [Bike Sharing sample application][bike-sharing-github] in Visual Studio Code. Open the Azure Kubernetes Service extension and select the **bikeapp** namespace in the **MyAKS** cluster. Right-click the **bikeapp** node, and choose **Use Namespace**.
+On your development computer, make sure your current context is set to the cluster and namespace in which your application is running.
 
-![Select Namespace](images/bridge-to-kubernetes-vs-code/select-namespace.png)
+Open the workspace for the app you want to debug in Visual Studio Code. Open the Command Palette (`kb(workbench.action.showCommands)`), and run the command **Bridge to Kubernetes: Configure** to start the configuration process.
 
-Use the `npm install` command to install the dependencies for the application.
+Choose the Kubernetes service you wish to redirect to your local version.
 
-```console
-npm install
-```
+![Select the service to connect to](images/minikube/select_service.png)
 
-Open the Command Palette (`kb(workbench.action.showCommands)`), and run the command **Bridge to Kubernetes: Configure** to start the configuration process.
-
-Choose the **bikes** service.
-
-![Choose Service](images/bridge-to-kubernetes-vs-code/choose-service.png)
-
-All traffic in the Kubernetes cluster is redirected for the bikes service to the version of your application running in your development computer. Bridge to Kubernetes also routes all outbound traffic from the application back to your Kubernetes cluster.
+All traffic in the Kubernetes cluster is redirected for your service to the version of your application running in your development computer. Bridge to Kubernetes also routes all outbound traffic from the application back to your Kubernetes cluster.
 
 > **Important**: You can only redirect services that have a single pod.
 
-After you select your service, you are prompted to enter the TCP port for your local application. For this example, enter "3000".
+After you select your service, you are prompted to enter the TCP port that your application uses to run locally.
 
-![Connect choose port](images/bridge-to-kubernetes-vs-code/choose-port.png)
+![Enter the port number](images/minikube/enter_port.png)
 
-Choose **Launch via NPM** as the launch task.
+Choose a debug profile that you normally use when running your application locally.
 
-![Connect choose launch task](images/bridge-to-kubernetes-vs-code/choose-launch.png)
+![Choose the debugger launch task](images/minikube/launch_task.png)
 
 > **Note**: You will be prompted to allow the **EndpointManager** to run elevated and modify your hosts file.
```

Comment:
Shouldn't this note be put further in the tutorial? The EndpointManager prompt will only happen when starting to debug using Bridge.